package maidsafe.encrypt.protobuf;

message ChunkDetails {
  required bytes hash = 1;
  required bytes pre_hash = 2;
  required uint32 size = 3;
  required uint32 pre_hash_state = 4;
}

message DataMap {
  repeated ChunkDetails chunk_details = 1;
  optional bytes content = 2;
}
